src/hooks/useRestMaxResultsCount.ts

```diff
@@ -0,0 +1,5 @@
+import useSystemSetting from './useSystemSetting';
+
+export default function useRestMaxResultsCount() {
+  return useSystemSetting('webservices.rest.maxResultsDefault');
+}
```

Comment:
I don't see a good reason for having an extra (this) hook. Can we just use the `useSystemSetting(setting)` hook directly?